app/views/appointments/new.html.erb

```diff
@@ -55,7 +55,34 @@
         <div class="appointment">
           <input type="submit" value="Submit">
         </div>
-      </form>
+      </form> -->
+      <%= form_for @appointment do |f| %>
```

Comment:
You shouldn't allow people who are not logged-in to see that form, otherwise they will spent 1 minute to fill-in the form and when they submit they move to the log in page and they've lost all their data! Instead you need to check if the user is logged in, if true you show the form, if false you show a message like "please log in to make an appointment", with a link to login/signup